### PR TITLE
SpeedInfo - fixed MUTCD 3-digit speed limit rendering

### PR DESCRIPTION
**Pull request by tomaszrybakiewicz**
Closes NAVAND-1056

### Description
 - fixed MUTCD 3-digit speed limit rendering in `MapboxSpeedInfoView`
- Added more customization testing options to the QA-Test-App > “Customized NavigationView” Activity

### Screenshots or Gifs

_Screen recording of the QA-Test-App captured on Pixel 6_

https://user-images.githubusercontent.com/2678039/215520363-1bb7c283-7c57-42e2-9e4b-bf0d5d843889.mp4

### CODE REVIEW

File: qa-test-app/src/main/res/layout/layout_drawer_menu_nav_view_custom.xml [hunk 1]
```diff
@@ -136,9 +136,42 @@
                     android:layout_height="wrap_content"
                     android:layout_weight="1"
                     android:paddingVertical="10dp"
-                    android:text="Show Speed Limit"
+                    android:text="Show Speed Info"
                     android:textAllCaps="true" />
 
+                <androidx.appcompat.widget.SwitchCompat
```
Comment (Zayankovsky) [inline qa-test-app/src/main/res/layout/layout_drawer_menu_nav_view_custom.xml:142]: There is the Speed limit section below, should we move the settings there?

File: libnavui-speedlimit/src/main/res/layout/mapbox_speed_info_view.xml [hunk 2]
```diff
@@ -44,12 +46,13 @@
                 android:layout_width="0dp"
                 android:layout_height="wrap_content"
                 android:gravity="center"
+                android:maxLines="1"
                 android:includeFontPadding="false"
                 app:layout_constraintEnd_toEndOf="parent"
                 app:layout_constraintStart_toStartOf="parent"
                 app:layout_constraintTop_toBottomOf="@id/postedSpeedLegend"
                 tools:textAppearance="@style/MapboxSpeedInfoPostedSpeedMutcdStyle"
```
Comment (Zayankovsky) [inline libnavui-speedlimit/src/main/res/layout/mapbox_speed_info_view.xml:54]: I see that the style defines text size as 32sp, while the width of the view is fixed and is specified in dp. Which means the speed value can easily not fit into the view if user increases text size in the system settings. That is how it looks for me (my current speed is actually 108, not 10):

If Drop-In doesn't support custom text sizes at the moment, then text size should be specified in dp.

Comment (tomaszrybakiewicz) [reply libnavui-speedlimit/src/main/res/layout/mapbox_speed_info_view.xml:54]: Good catch. Thx. Let me include that change as a part of this PR.

Comment (tomaszrybakiewicz) [reply libnavui-speedlimit/src/main/res/layout/mapbox_speed_info_view.xml:54]: Done

https://user-images.githubusercontent.com/2678039/215582268-90b10796-5e81-46ce-b13d-f3574cdbad5d.mp4

**Review by Zayankovsky — APPROVED** [review]

**Pull request merged**